backend/app/verification.py

```diff
@@ -233,6 +237,24 @@ class VerificationStatus(BaseModel):
     detail: str | None
 
 
+def _is_github_app(appid: str) -> bool:
+    """Determines whether the app is a repo in github.com/flathub."""
+    try:
+        gh = github.Github(
+            config.settings.github_client_id, config.settings.github_client_secret
+        )
+        try:
+            gh.get_repo(f"flathub/{appid}")
```

Comment:
Wondering, if we need to check `archived` here. And how the flow would work, to move from a github repo to our own, but keeping the appId the same.

Reply:
Checking and discarding `archived`, would at least allow apps, that already moved like firefox / obs to create their account. Still we somehow need to prevent others from doing that before the right person does it.

Reply:
Wouldn't that also allow people to revive an EOL'd app? Is that something we want to prevent?

Reply:
I think that would be fine, they verified the app first. So unless they somehow got hold of that url/git account, they should be able to revive an app.

I think we allow apps to be revived right now too, if there is a new maintainer, that takes over the flathub github repo.